internal/service/ec2/vpc_ipam_preview_next_cidr.go

```diff
@@ -0,0 +1,118 @@
+package ec2
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+	// "github.com/hashicorp/terraform-provider-aws/internal/flex"
+	// "github.com/hashicorp/terraform-provider-aws/internal/verify"
+)
+
+func ResourceVPCIpamPreviewNextCidr() *schema.Resource {
+	return &schema.Resource{
+		Create: ResourceVPCIpamPreviewNextCidrCreate,
+		Read:   ResourceVPCIpamPreviewNextCidrRead,
+		Delete: schema.Noop,
+		// Having imports results in errors
+		// 	error running import: exit status 1
+		// 	Error: Error allocating cidr from IPAM pool (): InvalidParameterValue: The allocation size is too big for the pool.
```

Comment:
we can remove notes about import in the resource and also in the acceptance test. because this resource does not actually produce anything, if we needed to import a "previewed" cidr then we wouldnt need the resource call in the first place and would just use the cidr we're attempting to import